File: python/create_directoryobject_from_group.py
import http.client

conn = http.client.HTTPSConnection("graph.microsoft.com")

payload = "{ }"

headers = { 'content-type': "application/json" }

conn.request("PUT", "/v1.0/users/%7Bid%7D/manager/$ref", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: node/create_directoryobject_from_group.js
var http = require("https");

var options = {
  "method": "PUT",
  "hostname": "graph.microsoft.com",
  "port": null,
  "path": "/v1.0/users/%7Bid%7D/manager/$ref",
  "headers": {
    "content-type": "application/json"
  }
};

var req = http.request(options, function (res) {
  var chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.write("{ }");
req.end();
